nx/include/switch/services/psm.h

```diff
@@ -38,8 +38,11 @@ void psmExit(void);
 Service* psmGetServiceSession(void);
 
 Result psmGetBatteryChargePercentage(u32 *out);
+Result psmGetRawBatteryChargePercentage(double *out);
+Result psmGetBatteryAgePercentage(double *out);
 Result psmGetChargerType(ChargerType *out);
 Result psmGetBatteryVoltageState(PsmBatteryVoltageState *out);
+Result psmIsEnoughPowerSupplied(u8 *out);
```

Comment:
This should be `bool *out`

Reply:
Is `bool` one byte in dkA64's ABI?

Reply:
Yes, it is in AAPCS64. Also note how other service wrappers tend to have xxOutBool meta-wrappers that explicitly do `value != 0`, you might want to add one here if there isn't already one.